compose/ui/ui-text/src/skikoTest/kotlin/androidx/compose/ui/text/SkikoParagraphTest.kt

```diff
@@ -324,10 +329,60 @@ class SkikoParagraphTest {
         )
     }
 
-    private fun simpleParagraph(text: String) = Paragraph(
+    @Test
+    fun getHorizontalPosition_cursor_empty_textfield_ltr_start_alignment() {
+        val paragraph = simpleParagraph("", TextStyle(textAlign = TextAlign.Start, textDirection = TextDirection.Ltr))
+        val cursorHorizontalPosition: Float = paragraph.getHorizontalPosition(0, false)
+        assertEquals(minWidthConstraint.toFloat(), cursorHorizontalPosition)
```

Comment:
minWidth != left. Why do you need this val at all?

Reply:
> minWidth != left.
haven't thought about that, thx
removed it